snyk-monitor/values.yaml

```diff
@@ -153,10 +153,11 @@ skopeo:
 
 workers:
   count: 5
+
 sysdig:
   enabled: false
-  namespace: sysdig-agent
-  secretName: sysdig-eve-secret
+  secretName: snyk-sysdig-secret
+  legacySecretName: sysdig-eve-secret
```

Comment:
this might break a customer's deployment if they have renamed their secret

Reply:
Auch, good point. Thinking about an alternative.